src/hooks/JB721Delegate/useNftAccountBalance.ts

```diff
@@ -10,23 +11,13 @@ export function useNftAccountBalance({
   dataSourceAddress: string | undefined
   accountAddress: string | undefined
 }) {
-  return useSubgraphQuery(
-    dataSourceAddress && accountAddress
-      ? {
-          entity: 'jb721DelegateToken',
-          keys: ['tokenId', 'address', 'tokenUri'],
-          where: [
-            {
-              key: 'address',
-              value: dataSourceAddress ?? '',
-            },
-            {
-              key: 'owner',
-              value: `{ wallet: "${accountAddress ?? ''}" }`,
-              nested: true,
-            },
-          ],
-        }
-      : null,
-  )
+  return useJb721DelegateTokensQuery({
+    client,
+    variables: {
+      where: {
+        ...(dataSourceAddress ? { address: dataSourceAddress } : {}),
+        ...(accountAddress ? { owner: accountAddress } : {}),
```

Comment:
wrapping this .graphql query in a hook makes it easier to apply conditional variables at the component end